jobs/rocoto/coupled_ic.sh

```diff
@@ -4,113 +4,130 @@ source "$HOMEgfs/ush/preamble.sh"
 
 ###############################################################
 ## Abstract:
-## Create FV3 initial conditions from GFS intitial conditions
-## RUN_ENVIR : runtime environment (emc | nco)
+## Copy initial conditions from BASE_CPLIC to ROTDIR for coupled forecast-only runs
 ## HOMEgfs   : /full/path/to/workflow
 ## EXPDIR : /full/path/to/config/files
-## CDATE  : current date (YYYYMMDDHH)
 ## CDUMP  : cycle name (gdas / gfs)
 ## PDY    : current date (YYYYMMDD)
 ## cyc    : current cycle (HH)
 ###############################################################
 
 ###############################################################
-# Source FV3GFS workflow modules
-. $HOMEgfs/ush/load_fv3gfs_modules.sh
+# Source workflow modules
+. "${HOMEgfs}/ush/load_fv3gfs_modules.sh"  # TODO: Do we need any modules for this job??
```

Comment:
> TODO: Do we need any modules for this job??

If you're not pulling from HPSS (hpss module) or using `ndate` for `GDATE` (or similar date calculations - prod_util module) then probably don't need to load any modules.

Reply:
Ok. I will remove.  There is no HPSS or prod_util use in this job